### PR TITLE
New strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**
This is my working list of things I am not sure about, and things to try to understand. Help welcomed.

1. "Enable clipboard notifications to copy fields when viewing an entry"
2. "chars" Could something else be used?
3. "don't vs do not" Both are in use.
4. "<string name="content_description_repeat_toggle_password_visibility">Repeat toggle password visibility</string>"
5. "autosave / autosaving database" (small) consistency issue
6. "(only in modifyable mode)" Is that part needed?
7. Is "next time" clear from the context of the presented string?
8. "<string name="max_history_items_summary">Limit the number of history items per entry</string>"
9. "<string name="biometric_unlock_enable_summary">Lets you scan your biometric to open the database</string>"
10. "Type the password before clicking the biometric button."
11. "entry save" ???
12. "at least one credential must be set"  what is the credential here?
13. "Password checkbox" I get what this is, but it isn't obvious what it does
14. "Keyfile checkbox" Likewise
15. "Node children" Almost no clue.

### CODE REVIEW

**Comment by J-Jamet** [comment]
1. Setting description to enable clipboard notifications
2. characters?
3. OK set everything to "do not"
4. Content description of the image view for the repeat password visibility icon
5. OK  "Autosaving"
6. Yes because the user might wonder if the "Write-protected" mode works in this case.
7. I don't know if it's clear, but it's copied from KeePass and I don't know how to say it otherwise.
8. Do what it says
9. Setting to activate biometric
10. Specifies what the user should do because a wrong action is performing.
11. Description of an image button, must be "Save entry"
12. Today, a password or a key file, but there will be other elements thereafter so we cannot say, we must at least select one.
13. Only a description of CompatCheckBox view to describe what the dialog does
14. Same as 13.
15. Content description of a recyclerview to describe a view in the screen 

Content description are useful for google code and better for screen algo recognition
I noticed weird elements in your commit:
A. "A file manager in needed to create" -> "A file manager is needed to create"
B. "(B)" -> maybe "(Byte)" To be more meaningful
C. "Recommend reneawl" -> "Recommend renewal"
D. Why ".gzip"? it's not a suffix for the file format but the compression of binaries
E. Shows \"Open\" button if no credentials are selected -> No, the button is always visible

For the other modifications, I don't see any worries, thanks for the work. I give you time to finish.

**Comment by comradekingu** [comment]
Good to go AFAIK.

7 unsolved
15 still unclear

D. "gzip" is what it is called. ".gzip" avoids the problem of it being either "Gzip" or "gzip". https://www.gzip.org
F. Anything I need to do?

Biometry/ic section has some strings in a different place, needs some work for another day.
Don't know if going with "shows instead of show (and similar) in descriptions. Probably not consistent.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
     <string name="beta_dontask">Don\'t show again</string>
     <string name="brackets">Brackets</string>
     <string name="extended_ASCII">Extended ASCII</string>
-    <string name="file_manager_install_description">Create, Open and Save a database file requires installing a file manager that accepts the Intent action ACTION_CREATE_DOCUMENT and ACTION_OPEN_DOCUMENT</string>
+    <string name="file_manager_install_description">A file manager in needed to create, open and save database files.</string>
```
[the remark this answers is not in the document]
Comment (J-Jamet) [reply app/src/main/res/values/strings.xml]: let "requires installing a file manager that accepts the Intent action ACTION_CREATE_DOCUMENT and ACTION_OPEN_DOCUMENT"

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -213,7 +213,7 @@
     <string name="rounds_explanation">Additional encryption rounds provide higher protection against brute force attacks, but can really slow down loading and saving.</string>
     <string name="rounds_hint">transformation rounds</string>
     <string name="memory_usage">Memory usage</string>
-    <string name="memory_usage_explanation">Amount of memory (in binary bytes) to be used by the key derivation function.</string>
+    <string name="memory_usage_explanation">Amount of memory (B) to be used by the key derivation function.</string>
```
Comment (J-Jamet) [inline app/src/main/res/values/strings.xml]: Change to "(Bytes)"

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -303,28 +303,28 @@
...
     <string name="recycle_bin_summary">Moves groups and entries to \"Recycle bin\" group before deleting</string>
-    <string name="recycle_bin_group_title">Recycle bin group</string>
+    <string name="recycle_bin_group_title">\"Recycle bin\" group</string>
     <string name="max_history_items_title">Max. history items</string>
-    <string name="max_history_items_summary">Limit number of history items per entry</string>
+    <string name="max_history_items_summary">Limit the number of history items per entry</string>
     <string name="max_history_size_title">Max. history size</string>
-    <string name="max_history_size_summary">Limit history size per entry (in binary bytes)</string>
-    <string name="settings_database_recommend_changing_master_key_title">Recommend changing</string>
+    <string name="max_history_size_summary">Limit history size per entry (B)</string>
```
Comment (J-Jamet) [inline app/src/main/res/values/strings.xml]: '(Bytes)"

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -339,12 +339,12 @@
     <string name="other">Other</string>
     <string name="compression">Compression</string>
     <string name="compression_none">None</string>
-    <string name="compression_gzip">GZip</string>
-    <string name="recycle_bin">Recycle Bin</string>
+    <string name="compression_gzip">.gzip</string>
```
Comment (J-Jamet) [inline app/src/main/res/values/strings.xml]: gzip

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -365,15 +365,15 @@
     <string name="keyboard_keys_category">Keys</string>
     <string name="keyboard_auto_go_action_title">Auto key action</string>
     <string name="keyboard_auto_go_action_summary">Action of the "Go" key performed automatically after pressing a "Field" key</string>
-    <string name="keyboard_key_vibrate_title">Vibrate on keypress</string>
-    <string name="keyboard_key_sound_title">Sound on keypress</string>
+    <string name="keyboard_key_vibrate_title">Vibratory keypresses</string>
+    <string name="keyboard_key_sound_title">Audible keypresses</string>
     <string name="allow_no_password_title">Allow no master key</string>
-    <string name="allow_no_password_summary">Enable the \"Open\" button if no credentials are selected</string>
+    <string name="allow_no_password_summary">Shows \"Open\" button if no credentials are selected</string>
```
Comment (J-Jamet) [inline app/src/main/res/values/strings.xml]: "Enable" is good

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -303,28 +303,28 @@
...
-    <string name="recycle_bin_group_title">Recycle bin group</string>
+    <string name="recycle_bin_group_title">\"Recycle bin\" group</string>
     <string name="max_history_items_title">Max. history items</string>
-    <string name="max_history_items_summary">Limit number of history items per entry</string>
+    <string name="max_history_items_summary">Limit the number of history items per entry</string>
     <string name="max_history_size_title">Max. history size</string>
-    <string name="max_history_size_summary">Limit history size per entry (in binary bytes)</string>
-    <string name="settings_database_recommend_changing_master_key_title">Recommend changing</string>
+    <string name="max_history_size_summary">Limit history size per entry (B)</string>
+    <string name="settings_database_recommend_changing_master_key_title">Recommend reneawl</string>
```
Comment (J-Jamet) [inline app/src/main/res/values/strings.xml]: "renewal"

File: app/src/main/res/values/strings.xml (hunk 4 above)
Comment (J-Jamet) [reply app/src/main/res/values/strings.xml]: "gzip" (what is in the string currently) is what it is called. ".gzip" avoids the problem of it being either "Gzip" or "gzip" ? Why it's a problem?

**Comment by J-Jamet** [comment]
Manually update string in a new commit.

**Pull request closed without merging**